src/couch/test/eunit/couchdb_attachments_tests.erl

```diff
@@ -102,13 +102,7 @@ attachments_md5_tests() ->
             fun teardown/1,
             [
                 fun should_upload_attachment_without_md5/1,
-                fun should_upload_attachment_by_chunks_without_md5/1,
-                fun should_upload_attachment_with_valid_md5_header/1,
-                fun should_upload_attachment_by_chunks_with_valid_md5_header/1,
-                fun should_upload_attachment_by_chunks_with_valid_md5_trailer/1,
-                fun should_reject_attachment_with_invalid_md5/1,
-                fun should_reject_chunked_attachment_with_invalid_md5/1,
-                fun should_reject_chunked_attachment_with_invalid_md5_trailer/1
```

Comment:
Should we assert what happens if users keep uploading stuff with md5 both valid and invalid ones?

Reply:
Good question. My view is that this PR (assuming it's right and complete) yields a CouchDB that completely ignores the Content-MD5 header, so it behaves identically whether the header is absent, present with the right value or present with the wrong value. Do we need tests for something that has no effect on CouchDB just because it did previously?

Reply:
I was mainly wondering about the trailer and if content md5 was the only trailer we ever accepted. As long we can ignore it and it won't mess up parsing of chunked requests that's fine, no need for extra tests.

Reply:
If I'd removed the (0, _Footers) clause it would have caused a problem, yes. any footer present would be interpreted as a chunk and that's no good.

Reply:
Good point, agree. 

Should we at least leave a comment when we consumer the _Footer to mention that at some point we used to accept trailers and we need that section to consume them, otherwise it would break the parsing logic? 

Maybe leave at least one test sending some footer junk (doesn't have to be MD5) in case this is the only place we actually tested that logic.

Reply:
sure.

Reply:
I've restored the original 'reject' tests and altered them to assert that those things now work (since we no longer support the Content-MD5 header).